test/conformance/helpers/channel_tracing_test_helper.go

```diff
@@ -138,7 +137,7 @@ func setupChannelTracingWithReply(
 		Span: tracinghelper.MatchHTTPSpanNoReply(
 			model.Server,
 			tracinghelper.WithHTTPHostAndPath(
-				fmt.Sprintf("%s-kn-channel.%s.svc.%s", channelName, client.Namespace, clusterDomain),
+				fmt.Sprintf("%s-kn-channel.%s.svc", channelName, client.Namespace),
```

Comment:
I see, the problem is that these are being used to match spans, and since the stuff on-cluster is sending the FQDN it no longer matches 🤔